awx/api/filters.py

```diff
@@ -65,7 +65,7 @@ def filter_queryset(self, request, queryset, view):
                 model = queryset.model
                 model_type = get_type_for_model(model)
                 if 'polymorphic_ctype' in get_all_field_names(model):
-                    types_pks = set([v for k,v in types_map.items() if k in types])
+                    types_pks = set([v for k,v in list(types_map.items()) if k in types])
```

Comment:
This could be shortened to `types_pks = set(types_map.values())`, for example:

```python
In [8]: set(d.values())                                                                                            
Out[8]: {1}

In [9]: set([v for k, v in d.items()])                                                                             
Out[9]: {1}
```